packages/plugin-workspace-tools/sources/commands/foreach.ts

```diff
@@ -56,53 +56,9 @@ const getWorkspaceDependenciesRecursive = (rootWorkspace: Workspace, project: Pr
 
 // eslint-disable-next-line arca/no-default-export
 export default class WorkspacesForeachCommand extends BaseCommand {
-  @Command.String()
-  commandName!: string;
-
-  @Command.Proxy()
-  args: Array<string> = [];
-
-  @Command.Boolean(`-R,--recursive`, {description: `Find packages via dependencies/devDependencies instead of using the workspaces field`})
-  recursive: boolean = false;
-
-  @Command.Boolean(`-A,--all`, {description: `Run the command on all workspaces of a project`})
-  all?: boolean;
-
-  @Command.Boolean(`-v,--verbose`, {description: `Prefix each output line with the name of the originating workspace`})
-  verbose: boolean = false;
-
-  @Command.Boolean(`-p,--parallel`, {description: `Run the commands in parallel`})
-  parallel: boolean = false;
-
-  @Command.Boolean(`-i,--interlaced`, {description: `Print the output of commands in real-time instead of buffering it`})
-  interlaced: boolean = false;
-
-  @Command.String(`-j,--jobs`, {description: `The maximum number of parallel tasks that the execution will be limited to`})
-  jobs?: number;
-
-  @Command.Boolean(`-t,--topological`, {description: `Run the command after all workspaces it depends on (regular) have finished`})
-  topological: boolean = false;
-
-  @Command.Boolean(`--topological-dev`, {description: `Run the command after all workspaces it depends on (regular + dev) have finished`})
-  topologicalDev: boolean = false;
-
-  @Command.Array(`--include`, {description: `An array of glob pattern idents; only matching workspaces will be traversed`})
-  include: Array<string> = [];
-
-  @Command.Array(`--exclude`, {description: `An array of glob pattern idents; matching workspaces won't be traversed`})
-  exclude: Array<string> = [];
-
-  @Command.Boolean(`--no-private`, {description: `Avoid running the command on private workspaces`})
-  publicOnly: boolean = false;
-
-  static schema = yup.object().shape({
-    jobs: yup.number().min(2),
-    parallel: yup.boolean().when(`jobs`, {
-      is: (val: number) => val > 1,
-      then: yup.boolean().oneOf([true], `--parallel must be set when using --jobs`),
-      otherwise: yup.boolean(),
-    }),
```

Comment:
Did you forget to port this part of the schema?

Reply:
Hm yep 🤔

Reply:
I'll skip it for now - `when` isn't implemented in Typanion yet, and there are some subtleties around how it should behave with regard to type inference.